src/System.Text.Json/tests/Serialization/DictionaryTests.cs

```diff
@@ -124,20 +124,42 @@ public static void InvalidJsonForValueShouldFail(Type type, string json)
             Assert.Throws<JsonException>(() => JsonSerializer.Parse(json, type));
         }
 
+        [Theory]
+        [InlineData(typeof(int[]), @"""test""")]
+        [InlineData(typeof(int[]), @"1")]
+        [InlineData(typeof(int[]), @"false")]
+        [InlineData(typeof(int[]), @"{}")]
+        [InlineData(typeof(int[]), @"[""test""")]
+        [InlineData(typeof(int[]), @"[true]")]
+        // [InlineData(typeof(int[]), @"[{}]")] TODO #38323: Uncomment when fixed
```

Comment:
This is pointing back to the current PR. Can you file an issue instead and add a link to that?